src/containers/departureBoard/DepartureBoard.jsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react'
+import EnturService from '@entur/sdk'
+import moment from 'moment'
+import Logo from '../../components/icons/Logo'
+import './styles.css'
+import { Bus, CityBike } from '../../components/icons'
+
+const service = new EnturService()
+
+const position = {
+    latitude: 59.903653,
+    longitude: 10.739232,
+}
+
+function getIcon(type, props) {
+    switch (type) {
+        case 'bus':
+            return <Bus color="#5AC39A" {...props} />
+        case 'bike':
+            return <CityBike {...props} />
+        default:
+            return null
+    }
+}
+
+class DepartureBoard extends React.Component {
+    state = {
+        stationData: [],
+        stopsData: [],
+    }
+
+    updateInterval = undefined
+
+    stopPlaceDepartures = () => {
+        const stops = this.state.stopsData
+        stops.forEach((stop, index) => {
+            service.getStopPlaceDepartures(stop.id, { onForBoarding: true, departures: 10 }).then(departures => {
+                const lineData = departures.map(departure => {
+                    const { expectedDepartureTime, destinationDisplay, serviceJourney } = departure
+                    const { line } = serviceJourney.journeyPattern
+                    const departureTime = moment(expectedDepartureTime)
+                    const minDiff = departureTime.diff(moment(), 'minutes')
+
+                    return {
+                        destination: destinationDisplay.frontText,
+                        type: line.transportMode,
+                        code: line.publicCode,
+                        time: minDiff < 15 ? (minDiff.toString() + 'min') : departureTime.format('HH:mm'),
+                    }
+                })
+                const newList = [...this.state.stopsData ]
+                newList[index].departures = lineData
+
+                this.setState({
+                    stopsData: newList,
+                })
+            })
+        })
+    }
+
+    updateTime = () => {
+        service.getBikeRentalStations(position, 200).then(stations => {
+            this.setState({
+                stationData: stations,
+            })
+        })
+        this.stopPlaceDepartures()
+    }
+
+    componentDidMount() {
+        service.getStopPlacesByPosition(position, 500).then(stops => {
+            const stopsData = stops.map(stop => {
+                return {
+                    ...stop,
+                    departures: [],
+                }
+            })
+            this.setState({ stopsData })
+            this.stopPlaceDepartures()
+            this.updateTime()
+        })
+        this.updateInterval = setInterval(this.updateTime, 10000)
+    }
+
+    componentWillUnmount() {
+        clearInterval(this.updateInterval)
+    }
+
+    renderList() {
```

Comment:
Reder list eller render render stop place? 
Trenger vel ikke dele funksjonen i to dersom man ikke bruker den til flere ting?